bucket/mremoteng-portable.json

```diff
@@ -0,0 +1,39 @@
+{
+    "homepage": "https://mremoteng.org/",
+    "description": "An open source, tabbed, multi-protocol, remote connections manager.",
+    "version": "1.76.20.24669",
+    "hash": "97b0e94bbdc409f3eb09b1017b3b58ccc0eae145a98a3bc6ea651ef1d5bc2d8e",
+    "license": "GPL-2.0-only",
+    "url": "https://github.com/mRemoteNG/mRemoteNG/releases/download/v1.76.20/mRemoteNG-Portable-1.76.20.24669.zip",
+    "bin": "mRemoteNG.exe",
+    "extract_dir": "",
+    "shortcuts": [
+        [
+            "mRemoteNG.exe",
+            "mRemoteNG"
+        ]
+    ],
+    "pre_install": [
+        "if (!(Test-Path \"$persist_dir\\mRemoteNG.settings\")) { New-Item \"$dir\\mRemoteNG.settings\" -ItemType File -Value \"<?xml version='1.0' encoding='utf-8'?>\"  | Out-Null }",
+        "if (!(Test-Path \"$persist_dir\\mRemoteNG.log\"  )) { New-Item \"$dir\\mRemoteNG.log\" -ItemType File | Out-Null }",
+        "if (!(Test-Path \"$persist_dir\\extApps.xml\"    )) { New-Item \"$dir\\extApps.xml\"   -ItemType File -Value \"<?xml version='1.0' encoding='utf-8'?>\"  | Out-Null }",
```

Comment:
What is this for?

Reply:
Default empty files to keep mRemoteNG happy and not failing (or loading empty data) which is not the default on a new installation. We can review thoroughly if needed.